### PR TITLE
Add keywords and descriptions to basics, intros and advanced guides

### PR DESCRIPTION
**Pull request by JerBouma**
I found out that if you Google something like `stocks openbb` you can't find us. This is most likely due to missing descriptions and keywords. This fixes that (hopefully).

### CODE REVIEW

**Comment by JerBouma** [comment]
@jose-donato Care to check if i did this right and approve?

File: website/content/sdk/guides/advanced/api-keys.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 title: Set API Keys
 sidebar_position: 1
+description: With the OpenBB SDK you can define your, often free, API key from various platforms like Alpha Vantage, FRED, IEX, Twitter, DeGiro, Binance and Coinglass. An API key is a set of random characters provided to you by a third party vendor that allows you to access data via their API endpoints. API keys can be managed directly from the SDK environment.
+keywords: [api, keys, api keys, data provider, data, free, alpha vantage, fred, iex, twitter, degiro, binance, coinglass, polygon, intrinio, openbb sdk]
```
Comment (jose-donato) [inline website/content/sdk/guides/advanced/api-keys.md:5]: iirc it should be 'api, keys, .....' and not []

**Review by jose-donato — CHANGES_REQUESTED** [review]
Some tips for the description:

Complement the title to make the snippet as appealing and click-worthy as possible.
Copy your meta description here if it makes sense. 
Keep it short and sweet. Facebook recommends 2-4 sentences, but that often truncates.

Keywords i left as a comment

File: website/content/sdk/guides/advanced/api-keys.md (hunk 1 above)
Comment (JerBouma) [reply website/content/sdk/guides/advanced/api-keys.md:5]: If I do that it gives me errors that it isn't an array.

image

Comment (JerBouma) [reply website/content/sdk/guides/advanced/api-keys.md:5]: Docs say it too: https://docusaurus.io/docs/next/seo

**Comment by JerBouma** [comment]
Care to approve so we can merge it in? @jose-donato. I believe everything is alright? I know it can be improved further but this is at least a step forward.

File: website/content/sdk/guides/advanced/api-keys.md (hunk 1 above)
Comment (jose-donato) [reply website/content/sdk/guides/advanced/api-keys.md:5]: oooh aight! i didnt think docusaurus was abstracting that. (that's the way it works on html)

**Pull request merged**